src/rust/ide/src/controller/project.rs

```diff
@@ -37,13 +39,14 @@ impl Handle {
     , language_server_client : language_server::Connection
     , language_server_binary : binary::Connection
     ) -> Self {
-        Handle {
-            module_registry     : default(),
-            parser              : Parser::new_or_panic(),
-            language_server_rpc : Rc::new(language_server_client),
-            language_server_bin : Rc::new(language_server_binary),
-            logger              : parent.sub("Project Controller"),
-        }
+        let module_registry         = default();
+        let parser                  = Parser::new_or_panic();
+        let language_server_rpc     = Rc::new(language_server_client);
+        let language_server_bin     = Rc::new(language_server_binary);
+        let logger                  = parent.sub("Project Controller");
+        let embedded_visualizations = default();
+        let visualization = Visualization::new(language_server_rpc.clone(),embedded_visualizations);
```

Comment:
alignment before `=`

Reply:
@mwu-tow I have followed the rule for keeping it one line instead of separating it in multiple lines.

Reply:
Extract `language_server_rpc.clone()` to variable `server_rpc` and you can apply Michals comment :)